Fail to compile on iMac (intel chip)

I failed to compile `flang` on an iMac with intel chip, for which `uname -a` returns
```
Darwin zMac 22.2.0 Darwin Kernel Version 22.2.0: Fri Nov 11 02:08:47 PST 2022; root:xnu-8792.61.2~4/RELEASE_X86_64 x86_64 i386 Darwin
```

Here is `classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeError.log`.

```
Checking whether the ASM compiler is GNU using "--version" did not match "(GNU assembler)|(GCC)|(Free Software Foundation)":
Apple clang version 14.0.0 (clang-1400.0.29.202)
Target: x86_64-apple-darwin22.2.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
Determining if the include file link.h exists failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-tQVUTn

Run Build Command(s):/usr/bin/make -f Makefile cmTC_138eb/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_138eb.dir/build.make CMakeFiles/cmTC_138eb.dir/build
Building C object CMakeFiles/cmTC_138eb.dir/CheckIncludeFile.c.o
/usr/bin/gcc   -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_138eb.dir/CheckIncludeFile.c.o -MF CMakeFiles/cmTC_138eb.dir/CheckIncludeFile.c.o.d -o CMakeFiles/cmTC_138eb.dir/CheckIncludeFile.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-tQVUTn/CheckIncludeFile.c
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-tQVUTn/CheckIncludeFile.c:1:10: fatal error: 'link.h' file not found
#include <link.h>
         ^~~~~~~~
1 error generated.
make[1]: *** [CMakeFiles/cmTC_138eb.dir/CheckIncludeFile.c.o] Error 1
make: *** [cmTC_138eb/fast] Error 2



Determining if the include file valgrind/valgrind.h exists failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-iGceue

Run Build Command(s):/usr/bin/make -f Makefile cmTC_fdcdb/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_fdcdb.dir/build.make CMakeFiles/cmTC_fdcdb.dir/build
Building C object CMakeFiles/cmTC_fdcdb.dir/CheckIncludeFile.c.o
/usr/bin/gcc   -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_fdcdb.dir/CheckIncludeFile.c.o -MF CMakeFiles/cmTC_fdcdb.dir/CheckIncludeFile.c.o.d -o CMakeFiles/cmTC_fdcdb.dir/CheckIncludeFile.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-iGceue/CheckIncludeFile.c
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-iGceue/CheckIncludeFile.c:1:10: fatal error: 'valgrind/valgrind.h' file not found
#include <valgrind/valgrind.h>
         ^~~~~~~~~~~~~~~~~~~~~
1 error generated.
make[1]: *** [CMakeFiles/cmTC_fdcdb.dir/CheckIncludeFile.c.o] Error 1
make: *** [cmTC_fdcdb/fast] Error 2



Determining if the include file CrashReporterClient.h exists failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-tKvpSg

Run Build Command(s):/usr/bin/make -f Makefile cmTC_688b9/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_688b9.dir/build.make CMakeFiles/cmTC_688b9.dir/build
Building C object CMakeFiles/cmTC_688b9.dir/CheckIncludeFile.c.o
/usr/bin/gcc   -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_688b9.dir/CheckIncludeFile.c.o -MF CMakeFiles/cmTC_688b9.dir/CheckIncludeFile.c.o.d -o CMakeFiles/cmTC_688b9.dir/CheckIncludeFile.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-tKvpSg/CheckIncludeFile.c
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-tKvpSg/CheckIncludeFile.c:1:10: fatal error: 'CrashReporterClient.h' file not found
#include <CrashReporterClient.h>
         ^~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make[1]: *** [CMakeFiles/cmTC_688b9.dir/CheckIncludeFile.c.o] Error 1
make: *** [cmTC_688b9/fast] Error 2



Determining if the function clock_gettime exists in the rt failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-NJmVxD

Run Build Command(s):/usr/bin/make -f Makefile cmTC_d7bd7/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_d7bd7.dir/build.make CMakeFiles/cmTC_d7bd7.dir/build
Building C object CMakeFiles/cmTC_d7bd7.dir/CheckFunctionExists.c.o
/usr/bin/gcc   -DCHECK_FUNCTION_EXISTS=clock_gettime -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_d7bd7.dir/CheckFunctionExists.c.o -MF CMakeFiles/cmTC_d7bd7.dir/CheckFunctionExists.c.o.d -o CMakeFiles/cmTC_d7bd7.dir/CheckFunctionExists.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-NJmVxD/CheckFunctionExists.c
Linking C executable cmTC_d7bd7
/usr/local/Cellar/cmake/3.25.1/bin/cmake -E cmake_link_script CMakeFiles/cmTC_d7bd7.dir/link.txt --verbose=1
/usr/bin/gcc  -DCHECK_FUNCTION_EXISTS=clock_gettime -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names CMakeFiles/cmTC_d7bd7.dir/CheckFunctionExists.c.o -o cmTC_d7bd7  -lrt 
ld: library not found for -lrt
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [cmTC_d7bd7] Error 1
make: *** [cmTC_d7bd7/fast] Error 2



Determining if the function pfm_initialize exists in the pfm failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-xK042t

Run Build Command(s):/usr/bin/make -f Makefile cmTC_d46da/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_d46da.dir/build.make CMakeFiles/cmTC_d46da.dir/build
Building C object CMakeFiles/cmTC_d46da.dir/CheckFunctionExists.c.o
/usr/bin/gcc   -DCHECK_FUNCTION_EXISTS=pfm_initialize -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_d46da.dir/CheckFunctionExists.c.o -MF CMakeFiles/cmTC_d46da.dir/CheckFunctionExists.c.o.d -o CMakeFiles/cmTC_d46da.dir/CheckFunctionExists.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-xK042t/CheckFunctionExists.c
Linking C executable cmTC_d46da
/usr/local/Cellar/cmake/3.25.1/bin/cmake -E cmake_link_script CMakeFiles/cmTC_d46da.dir/link.txt --verbose=1
/usr/bin/gcc  -DCHECK_FUNCTION_EXISTS=pfm_initialize -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names CMakeFiles/cmTC_d46da.dir/CheckFunctionExists.c.o -o cmTC_d46da  -lpfm 
ld: library not found for -lpfm
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [cmTC_d46da] Error 1
make: *** [cmTC_d46da/fast] Error 2



Determining if the lseek64 exist failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-iuDxzD

Run Build Command(s):/usr/bin/make -f Makefile cmTC_96008/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_96008.dir/build.make CMakeFiles/cmTC_96008.dir/build
Building C object CMakeFiles/cmTC_96008.dir/CheckSymbolExists.c.o
/usr/bin/gcc -D_LARGEFILE64_SOURCE  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_96008.dir/CheckSymbolExists.c.o -MF CMakeFiles/cmTC_96008.dir/CheckSymbolExists.c.o.d -o CMakeFiles/cmTC_96008.dir/CheckSymbolExists.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-iuDxzD/CheckSymbolExists.c
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-iuDxzD/CheckSymbolExists.c:9:19: error: use of undeclared identifier 'lseek64'; did you mean 'lseek'?
  return ((int*)(&lseek64))[argc];
                  ^~~~~~~
                  lseek
/Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk/usr/include/unistd.h:465:8: note: 'lseek' declared here
off_t    lseek(int, off_t, int);
         ^
1 error generated.
make[1]: *** [CMakeFiles/cmTC_96008.dir/CheckSymbolExists.c.o] Error 1
make: *** [cmTC_96008/fast] Error 2


File CheckSymbolExists.c:
/* */
#include <sys/types.h>
#include <unistd.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef lseek64
  return ((int*)(&lseek64))[argc];
#else
  (void)argc;
  return 0;
#endif
}
Determining if the mallctl exist failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-cAxOtk

Run Build Command(s):/usr/bin/make -f Makefile cmTC_85cae/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_85cae.dir/build.make CMakeFiles/cmTC_85cae.dir/build
Building C object CMakeFiles/cmTC_85cae.dir/CheckSymbolExists.c.o
/usr/bin/gcc   -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_85cae.dir/CheckSymbolExists.c.o -MF CMakeFiles/cmTC_85cae.dir/CheckSymbolExists.c.o.d -o CMakeFiles/cmTC_85cae.dir/CheckSymbolExists.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-cAxOtk/CheckSymbolExists.c
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-cAxOtk/CheckSymbolExists.c:2:10: fatal error: 'malloc_np.h' file not found
#include <malloc_np.h>
         ^~~~~~~~~~~~~
1 error generated.
make[1]: *** [CMakeFiles/cmTC_85cae.dir/CheckSymbolExists.c.o] Error 1
make: *** [cmTC_85cae/fast] Error 2


File CheckSymbolExists.c:
/* */
#include <malloc_np.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef mallctl
  return ((int*)(&mallctl))[argc];
#else
  (void)argc;
  return 0;
#endif
}
Determining if the mallinfo exist failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-byOruc

Run Build Command(s):/usr/bin/make -f Makefile cmTC_e2e31/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_e2e31.dir/build.make CMakeFiles/cmTC_e2e31.dir/build
Building C object CMakeFiles/cmTC_e2e31.dir/CheckSymbolExists.c.o
/usr/bin/gcc   -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_e2e31.dir/CheckSymbolExists.c.o -MF CMakeFiles/cmTC_e2e31.dir/CheckSymbolExists.c.o.d -o CMakeFiles/cmTC_e2e31.dir/CheckSymbolExists.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-byOruc/CheckSymbolExists.c
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-byOruc/CheckSymbolExists.c:2:10: fatal error: 'malloc.h' file not found
#include <malloc.h>
         ^~~~~~~~~~
1 error generated.
make[1]: *** [CMakeFiles/cmTC_e2e31.dir/CheckSymbolExists.c.o] Error 1
make: *** [cmTC_e2e31/fast] Error 2


File CheckSymbolExists.c:
/* */
#include <malloc.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef mallinfo
  return ((int*)(&mallinfo))[argc];
#else
  (void)argc;
  return 0;
#endif
}
Determining if the mallinfo2 exist failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-YrFkm3

Run Build Command(s):/usr/bin/make -f Makefile cmTC_ee6a2/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_ee6a2.dir/build.make CMakeFiles/cmTC_ee6a2.dir/build
Building C object CMakeFiles/cmTC_ee6a2.dir/CheckSymbolExists.c.o
/usr/bin/gcc   -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_ee6a2.dir/CheckSymbolExists.c.o -MF CMakeFiles/cmTC_ee6a2.dir/CheckSymbolExists.c.o.d -o CMakeFiles/cmTC_ee6a2.dir/CheckSymbolExists.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-YrFkm3/CheckSymbolExists.c
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-YrFkm3/CheckSymbolExists.c:2:10: fatal error: 'malloc.h' file not found
#include <malloc.h>
         ^~~~~~~~~~
1 error generated.
make[1]: *** [CMakeFiles/cmTC_ee6a2.dir/CheckSymbolExists.c.o] Error 1
make: *** [cmTC_ee6a2/fast] Error 2


File CheckSymbolExists.c:
/* */
#include <malloc.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef mallinfo2
  return ((int*)(&mallinfo2))[argc];
#else
  (void)argc;
  return 0;
#endif
}
Determining if the strerror_s exist failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-nKHefK

Run Build Command(s):/usr/bin/make -f Makefile cmTC_0ae5a/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_0ae5a.dir/build.make CMakeFiles/cmTC_0ae5a.dir/build
Building C object CMakeFiles/cmTC_0ae5a.dir/CheckSymbolExists.c.o
/usr/bin/gcc   -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_0ae5a.dir/CheckSymbolExists.c.o -MF CMakeFiles/cmTC_0ae5a.dir/CheckSymbolExists.c.o.d -o CMakeFiles/cmTC_0ae5a.dir/CheckSymbolExists.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-nKHefK/CheckSymbolExists.c
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-nKHefK/CheckSymbolExists.c:8:19: error: use of undeclared identifier 'strerror_s'; did you mean 'strerror_r'?
  return ((int*)(&strerror_s))[argc];
                  ^~~~~~~~~~
                  strerror_r
/Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk/usr/include/string.h:116:6: note: 'strerror_r' declared here
int      strerror_r(int __errnum, char *__strerrbuf, size_t __buflen);
         ^
1 error generated.
make[1]: *** [CMakeFiles/cmTC_0ae5a.dir/CheckSymbolExists.c.o] Error 1
make: *** [cmTC_0ae5a/fast] Error 2


File CheckSymbolExists.c:
/* */
#include <string.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef strerror_s
  return ((int*)(&strerror_s))[argc];
#else
  (void)argc;
  return 0;
#endif
}
Performing C SOURCE FILE Test HAVE_STRUCT_STAT_ST_MTIM_TV_NSEC failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-b2CpPR

Run Build Command(s):/usr/bin/make -f Makefile cmTC_dfd19/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_dfd19.dir/build.make CMakeFiles/cmTC_dfd19.dir/build
Building C object CMakeFiles/cmTC_dfd19.dir/src.c.o
/usr/bin/gcc -DHAVE_STRUCT_STAT_ST_MTIM_TV_NSEC  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_dfd19.dir/src.c.o -MF CMakeFiles/cmTC_dfd19.dir/src.c.o.d -o CMakeFiles/cmTC_dfd19.dir/src.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-b2CpPR/src.c
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-b2CpPR/src.c:7:36: error: no member named 'st_mtim' in 'struct stat'
  (void)sizeof(((struct stat *)0)->st_mtim.tv_nsec);
               ~~~~~~~~~~~~~~~~~~  ^
1 error generated.
make[1]: *** [CMakeFiles/cmTC_dfd19.dir/src.c.o] Error 1
make: *** [cmTC_dfd19/fast] Error 2


Source file was:

#include <sys/types.h>
#include <sys/stat.h>

int main()
{
  (void)sizeof(((struct stat *)0)->st_mtim.tv_nsec);
  return 0;
}


Determining if the __GLIBC__ exist failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-zkTzQk

Run Build Command(s):/usr/bin/make -f Makefile cmTC_4d6be/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_4d6be.dir/build.make CMakeFiles/cmTC_4d6be.dir/build
Building C object CMakeFiles/cmTC_4d6be.dir/CheckSymbolExists.c.o
/usr/bin/gcc   -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_4d6be.dir/CheckSymbolExists.c.o -MF CMakeFiles/cmTC_4d6be.dir/CheckSymbolExists.c.o.d -o CMakeFiles/cmTC_4d6be.dir/CheckSymbolExists.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-zkTzQk/CheckSymbolExists.c
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-zkTzQk/CheckSymbolExists.c:8:19: error: use of undeclared identifier '__GLIBC__'
  return ((int*)(&__GLIBC__))[argc];
                  ^
1 error generated.
make[1]: *** [CMakeFiles/cmTC_4d6be.dir/CheckSymbolExists.c.o] Error 1
make: *** [cmTC_4d6be/fast] Error 2


File CheckSymbolExists.c:
/* */
#include <stdio.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef __GLIBC__
  return ((int*)(&__GLIBC__))[argc];
#else
  (void)argc;
  return 0;
#endif
}
Performing C SOURCE FILE Test C_SUPPORTS_FNO_SEMANTIC_INTERPOSITION failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-lZU4Mo

Run Build Command(s):/usr/bin/make -f Makefile cmTC_d052a/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_d052a.dir/build.make CMakeFiles/cmTC_d052a.dir/build
Building C object CMakeFiles/cmTC_d052a.dir/src.c.o
/usr/bin/gcc -DC_SUPPORTS_FNO_SEMANTIC_INTERPOSITION  -fPIC  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk   -Werror -fno-semantic-interposition -MD -MT CMakeFiles/cmTC_d052a.dir/src.c.o -MF CMakeFiles/cmTC_d052a.dir/src.c.o.d -o CMakeFiles/cmTC_d052a.dir/src.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-lZU4Mo/src.c
clang: error: argument unused during compilation: '-fno-semantic-interposition' [-Werror,-Wunused-command-line-argument]
make[1]: *** [CMakeFiles/cmTC_d052a.dir/src.c.o] Error 1
make: *** [cmTC_d052a/fast] Error 2


Source file was:
int main(void) { return 0; }

Performing C++ SOURCE FILE Test CXX_SUPPORTS_FNO_SEMANTIC_INTERPOSITION failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-w34Yf5

Run Build Command(s):/usr/bin/make -f Makefile cmTC_08e0e/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_08e0e.dir/build.make CMakeFiles/cmTC_08e0e.dir/build
Building CXX object CMakeFiles/cmTC_08e0e.dir/src.cxx.o
/usr/bin/g++ -DCXX_SUPPORTS_FNO_SEMANTIC_INTERPOSITION  -fPIC  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk   -Werror -fno-semantic-interposition -std=c++14 -MD -MT CMakeFiles/cmTC_08e0e.dir/src.cxx.o -MF CMakeFiles/cmTC_08e0e.dir/src.cxx.o.d -o CMakeFiles/cmTC_08e0e.dir/src.cxx.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-w34Yf5/src.cxx
clang: error: argument unused during compilation: '-fno-semantic-interposition' [-Werror,-Wunused-command-line-argument]
make[1]: *** [CMakeFiles/cmTC_08e0e.dir/src.cxx.o] Error 1
make: *** [cmTC_08e0e/fast] Error 2


Source file was:
int main() { return 0; }

Performing C++ SOURCE FILE Test CXX_SUPPORTS_CLASS_MEMACCESS_FLAG failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-mavVkp

Run Build Command(s):/usr/bin/make -f Makefile cmTC_2acc0/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_2acc0.dir/build.make CMakeFiles/cmTC_2acc0.dir/build
Building CXX object CMakeFiles/cmTC_2acc0.dir/src.cxx.o
/usr/bin/g++ -DCXX_SUPPORTS_CLASS_MEMACCESS_FLAG  -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk   -Wclass-memaccess -std=c++14 -MD -MT CMakeFiles/cmTC_2acc0.dir/src.cxx.o -MF CMakeFiles/cmTC_2acc0.dir/src.cxx.o.d -o CMakeFiles/cmTC_2acc0.dir/src.cxx.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-mavVkp/src.cxx
warning: unknown warning option '-Wclass-memaccess'; did you mean '-Wclass-varargs'? [-Wunknown-warning-option]
1 warning generated.
Linking CXX executable cmTC_2acc0
/usr/local/Cellar/cmake/3.25.1/bin/cmake -E cmake_link_script CMakeFiles/cmTC_2acc0.dir/link.txt --verbose=1
/usr/bin/g++  -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names CMakeFiles/cmTC_2acc0.dir/src.cxx.o -o cmTC_2acc0 


Source file was:
int main() { return 0; }

Performing C++ SOURCE FILE Test LLVM_HAS_NOGLOBAL_CTOR_MUTEX failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-y8NYuS

Run Build Command(s):/usr/bin/make -f Makefile cmTC_9f511/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_9f511.dir/build.make CMakeFiles/cmTC_9f511.dir/build
Building CXX object CMakeFiles/cmTC_9f511.dir/src.cxx.o
/usr/bin/g++ -DLLVM_HAS_NOGLOBAL_CTOR_MUTEX  -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wmisleading-indentation -Werror=global-constructors  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -std=c++14 -MD -MT CMakeFiles/cmTC_9f511.dir/src.cxx.o -MF CMakeFiles/cmTC_9f511.dir/src.cxx.o.d -o CMakeFiles/cmTC_9f511.dir/src.cxx.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-y8NYuS/src.cxx
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-y8NYuS/src.cxx:3:21: error: declaration requires a global destructor [-Werror,-Wglobal-constructors]
  static std::mutex TestGlobalCtorDtor;
                    ^
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-y8NYuS/src.cxx:4:31: error: declaration requires a global destructor [-Werror,-Wglobal-constructors]
  static std::recursive_mutex TestGlobalCtorDtor2;
                              ^
2 errors generated.
make[1]: *** [CMakeFiles/cmTC_9f511.dir/src.cxx.o] Error 1
make: *** [cmTC_9f511/fast] Error 2


Source file was:

  #include <mutex>
  static std::mutex TestGlobalCtorDtor;
  static std::recursive_mutex TestGlobalCtorDtor2;
  int main() { (void)TestGlobalCtorDtor; (void)TestGlobalCtorDtor2; return 0;}
  

Performing C++ SOURCE FILE Test OPENMP_HAVE_WNO_MAYBE_UNINITIALIZED_FLAG failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-FPNcsG

Run Build Command(s):/usr/bin/make -f Makefile cmTC_6b89e/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_6b89e.dir/build.make CMakeFiles/cmTC_6b89e.dir/build
Building CXX object CMakeFiles/cmTC_6b89e.dir/src.cxx.o
/usr/bin/g++ -DOPENMP_HAVE_WNO_MAYBE_UNINITIALIZED_FLAG  -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wmisleading-indentation  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk   -Wno-maybe-uninitialized -std=c++14 -MD -MT CMakeFiles/cmTC_6b89e.dir/src.cxx.o -MF CMakeFiles/cmTC_6b89e.dir/src.cxx.o.d -o CMakeFiles/cmTC_6b89e.dir/src.cxx.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-FPNcsG/src.cxx
warning: unknown warning option '-Wno-maybe-uninitialized'; did you mean '-Wno-uninitialized'? [-Wunknown-warning-option]
1 warning generated.
Linking CXX executable cmTC_6b89e
/usr/local/Cellar/cmake/3.25.1/bin/cmake -E cmake_link_script CMakeFiles/cmTC_6b89e.dir/link.txt --verbose=1
/usr/bin/g++  -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wmisleading-indentation  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names CMakeFiles/cmTC_6b89e.dir/src.cxx.o -o cmTC_6b89e 


Source file was:
int main() { return 0; }

Performing C++ SOURCE FILE Test LIBOMP_HAVE_WNO_CLASS_MEMACCESS_FLAG failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-6Upuxm

Run Build Command(s):/usr/bin/make -f Makefile cmTC_2f8ec/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_2f8ec.dir/build.make CMakeFiles/cmTC_2f8ec.dir/build
Building CXX object CMakeFiles/cmTC_2f8ec.dir/src.cxx.o
/usr/bin/g++ -DLIBOMP_HAVE_WNO_CLASS_MEMACCESS_FLAG  -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wmisleading-indentation -Wall -Wcast-qual -Wformat-pedantic -Wimplicit-fallthrough -Wsign-compare -Wno-extra -Wno-pedantic -std=c++14  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk   -Wno-class-memaccess -std=c++14 -MD -MT CMakeFiles/cmTC_2f8ec.dir/src.cxx.o -MF CMakeFiles/cmTC_2f8ec.dir/src.cxx.o.d -o CMakeFiles/cmTC_2f8ec.dir/src.cxx.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-6Upuxm/src.cxx
warning: unknown warning option '-Wno-class-memaccess'; did you mean '-Wno-class-varargs'? [-Wunknown-warning-option]
1 warning generated.
Linking CXX executable cmTC_2f8ec
/usr/local/Cellar/cmake/3.25.1/bin/cmake -E cmake_link_script CMakeFiles/cmTC_2f8ec.dir/link.txt --verbose=1
/usr/bin/g++  -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wmisleading-indentation -Wall -Wcast-qual -Wformat-pedantic -Wimplicit-fallthrough -Wsign-compare -Wno-extra -Wno-pedantic -std=c++14  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names CMakeFiles/cmTC_2f8ec.dir/src.cxx.o -o cmTC_2f8ec 


Source file was:
int main() { return 0; }

Performing C++ SOURCE FILE Test LIBOMP_HAVE_WSTRINGOP_OVERFLOW_FLAG failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-VO8NUx

Run Build Command(s):/usr/bin/make -f Makefile cmTC_e5f2d/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_e5f2d.dir/build.make CMakeFiles/cmTC_e5f2d.dir/build
Building CXX object CMakeFiles/cmTC_e5f2d.dir/src.cxx.o
/usr/bin/g++ -DLIBOMP_HAVE_WSTRINGOP_OVERFLOW_FLAG  -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wmisleading-indentation -Wall -Wcast-qual -Wformat-pedantic -Wimplicit-fallthrough -Wsign-compare -Wno-extra -Wno-pedantic -std=c++14  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk   -Wstringop-overflow=0 -std=c++14 -MD -MT CMakeFiles/cmTC_e5f2d.dir/src.cxx.o -MF CMakeFiles/cmTC_e5f2d.dir/src.cxx.o.d -o CMakeFiles/cmTC_e5f2d.dir/src.cxx.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-VO8NUx/src.cxx
warning: unknown warning option '-Wstringop-overflow=0'; did you mean '-Wshift-overflow'? [-Wunknown-warning-option]
1 warning generated.
Linking CXX executable cmTC_e5f2d
/usr/local/Cellar/cmake/3.25.1/bin/cmake -E cmake_link_script CMakeFiles/cmTC_e5f2d.dir/link.txt --verbose=1
/usr/bin/g++  -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wmisleading-indentation -Wall -Wcast-qual -Wformat-pedantic -Wimplicit-fallthrough -Wsign-compare -Wno-extra -Wno-pedantic -std=c++14  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names CMakeFiles/cmTC_e5f2d.dir/src.cxx.o -o cmTC_e5f2d 


Source file was:
int main() { return 0; }

Performing C++ SOURCE FILE Test LIBOMP_HAVE_WNO_STRINGOP_TRUNCATION_FLAG failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-xah2Tz

Run Build Command(s):/usr/bin/make -f Makefile cmTC_09bd5/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_09bd5.dir/build.make CMakeFiles/cmTC_09bd5.dir/build
Building CXX object CMakeFiles/cmTC_09bd5.dir/src.cxx.o
/usr/bin/g++ -DLIBOMP_HAVE_WNO_STRINGOP_TRUNCATION_FLAG  -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wmisleading-indentation -Wall -Wcast-qual -Wformat-pedantic -Wimplicit-fallthrough -Wsign-compare -Wno-extra -Wno-pedantic -std=c++14  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk   -Wno-stringop-truncation -std=c++14 -MD -MT CMakeFiles/cmTC_09bd5.dir/src.cxx.o -MF CMakeFiles/cmTC_09bd5.dir/src.cxx.o.d -o CMakeFiles/cmTC_09bd5.dir/src.cxx.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-xah2Tz/src.cxx
warning: unknown warning option '-Wno-stringop-truncation'; did you mean '-Wno-string-concatenation'? [-Wunknown-warning-option]
1 warning generated.
Linking CXX executable cmTC_09bd5
/usr/local/Cellar/cmake/3.25.1/bin/cmake -E cmake_link_script CMakeFiles/cmTC_09bd5.dir/link.txt --verbose=1
/usr/bin/g++  -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wmisleading-indentation -Wall -Wcast-qual -Wformat-pedantic -Wimplicit-fallthrough -Wsign-compare -Wno-extra -Wno-pedantic -std=c++14  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names CMakeFiles/cmTC_09bd5.dir/src.cxx.o -o cmTC_09bd5 


Source file was:
int main() { return 0; }

Performing C SOURCE FILE Test LIBOMP_HAVE_MMIC_FLAG failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-b9WAdz

Run Build Command(s):/usr/bin/make -f Makefile cmTC_df238/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_df238.dir/build.make CMakeFiles/cmTC_df238.dir/build
Building C object CMakeFiles/cmTC_df238.dir/src.c.o
/usr/bin/gcc -DLIBOMP_HAVE_MMIC_FLAG  -fPIC -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wstring-conversion -Wmisleading-indentation -Wall -Wcast-qual -Wformat-pedantic -Wimplicit-fallthrough -Wsign-compare -Wno-extra -Wno-pedantic -mmic -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk   -mmic -MD -MT CMakeFiles/cmTC_df238.dir/src.c.o -MF CMakeFiles/cmTC_df238.dir/src.c.o.d -o CMakeFiles/cmTC_df238.dir/src.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-b9WAdz/src.c
clang: error: unknown argument: '-mmic'
clang: error: unknown argument: '-mmic'
make[1]: *** [CMakeFiles/cmTC_df238.dir/src.c.o] Error 1
make: *** [cmTC_df238/fast] Error 2


Source file was:
int main(void) { return 0; }

Performing C SOURCE FILE Test LIBOMP_HAVE_M32_FLAG failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-UvV30Y

Run Build Command(s):/usr/bin/make -f Makefile cmTC_78aab/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_78aab.dir/build.make CMakeFiles/cmTC_78aab.dir/build
Building C object CMakeFiles/cmTC_78aab.dir/src.c.o
/usr/bin/gcc -DLIBOMP_HAVE_M32_FLAG  -fPIC -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wstring-conversion -Wmisleading-indentation -Wall -Wcast-qual -Wformat-pedantic -Wimplicit-fallthrough -Wsign-compare -Wno-extra -Wno-pedantic -m32 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk   -m32 -MD -MT CMakeFiles/cmTC_78aab.dir/src.c.o -MF CMakeFiles/cmTC_78aab.dir/src.c.o.d -o CMakeFiles/cmTC_78aab.dir/src.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-UvV30Y/src.c
Linking C executable cmTC_78aab
/usr/local/Cellar/cmake/3.25.1/bin/cmake -E cmake_link_script CMakeFiles/cmTC_78aab.dir/link.txt --verbose=1
/usr/bin/gcc  -fPIC -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wstring-conversion -Wmisleading-indentation -Wall -Wcast-qual -Wformat-pedantic -Wimplicit-fallthrough -Wsign-compare -Wno-extra -Wno-pedantic -m32 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names CMakeFiles/cmTC_78aab.dir/src.c.o -o cmTC_78aab 
ld: warning: The i386 architecture is deprecated for macOS (remove from the Xcode build setting: ARCHS)
ld: warning: ignoring file /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk/usr/lib/libSystem.tbd, missing required architecture i386 in file /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk/usr/lib/libSystem.tbd (3 slices)
ld: dynamic executables or dylibs must link with libSystem.dylib for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [cmTC_78aab] Error 1
make: *** [cmTC_78aab/fast] Error 2


Source file was:
int main(void) { return 0; }

Determining if the _aligned_malloc exist failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-y7NH5Y

Run Build Command(s):/usr/bin/make -f Makefile cmTC_87188/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_87188.dir/build.make CMakeFiles/cmTC_87188.dir/build
Building C object CMakeFiles/cmTC_87188.dir/CheckSymbolExists.c.o
/usr/bin/gcc   -fPIC  -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wstring-conversion -Wmisleading-indentation -Wall -Wcast-qual -Wformat-pedantic -Wimplicit-fallthrough -Wsign-compare -Wno-extra -Wno-pedantic  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_87188.dir/CheckSymbolExists.c.o -MF CMakeFiles/cmTC_87188.dir/CheckSymbolExists.c.o.d -o CMakeFiles/cmTC_87188.dir/CheckSymbolExists.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-y7NH5Y/CheckSymbolExists.c
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-y7NH5Y/CheckSymbolExists.c:2:10: fatal error: 'malloc.h' file not found
#include <malloc.h>
         ^~~~~~~~~~
1 error generated.
make[1]: *** [CMakeFiles/cmTC_87188.dir/CheckSymbolExists.c.o] Error 1
make: *** [cmTC_87188/fast] Error 2


File CheckSymbolExists.c:
/* */
#include <malloc.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef _aligned_malloc
  return ((int*)(&_aligned_malloc))[argc];
#else
  (void)argc;
  return 0;
#endif
}
Performing C SOURCE FILE Test LIBOMP_HAVE_VERSION_SYMBOLS failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-LScbgc

Run Build Command(s):/usr/bin/make -f Makefile cmTC_b8f87/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_b8f87.dir/build.make CMakeFiles/cmTC_b8f87.dir/build
Building C object CMakeFiles/cmTC_b8f87.dir/src.c.o
/usr/bin/gcc -DLIBOMP_HAVE_VERSION_SYMBOLS  -fPIC -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wstring-conversion -Wmisleading-indentation -Wall -Wcast-qual -Wformat-pedantic -Wimplicit-fallthrough -Wsign-compare -Wno-extra -Wno-pedantic -Wl,--version-script=/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/projects/openmp/runtime/__version_script.txt -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_b8f87.dir/src.c.o -MF CMakeFiles/cmTC_b8f87.dir/src.c.o.d -o CMakeFiles/cmTC_b8f87.dir/src.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-LScbgc/src.c
clang: warning: -Wl,--version-script=/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/projects/openmp/runtime/__version_script.txt: 'linker' input unused [-Wunused-command-line-argument]
<inline asm>:1:1: error: unknown directive
.symver func1, func@VER1
^
<inline asm>:2:1: error: unknown directive
.symver func2, func@VER2
^
2 errors generated.
make[1]: *** [CMakeFiles/cmTC_b8f87.dir/src.c.o] Error 1
make: *** [cmTC_b8f87/fast] Error 2


Source file was:
#include <stdio.h>
    void func1() { printf("Hello"); }
    void func2() { printf("World"); }
    __asm__(".symver func1, func@VER1");
    __asm__(".symver func2, func@VER2");
    int main() {
      func1();
      func2();
      return 0;
    }

Performing C SOURCE FILE Test LIBOMP_HAVE_PSAPI failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-4SXScv

Run Build Command(s):/usr/bin/make -f Makefile cmTC_fa172/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_fa172.dir/build.make CMakeFiles/cmTC_fa172.dir/build
Building C object CMakeFiles/cmTC_fa172.dir/src.c.o
/usr/bin/gcc -DLIBOMP_HAVE_PSAPI  -fPIC -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wstring-conversion -Wmisleading-indentation -Wall -Wcast-qual -Wformat-pedantic -Wimplicit-fallthrough -Wsign-compare -Wno-extra -Wno-pedantic  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_fa172.dir/src.c.o -MF CMakeFiles/cmTC_fa172.dir/src.c.o.d -o CMakeFiles/cmTC_fa172.dir/src.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-4SXScv/src.c
/tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-4SXScv/src.c:1:10: fatal error: 'windows.h' file not found
#include <windows.h>
         ^~~~~~~~~~~
1 error generated.
make[1]: *** [CMakeFiles/cmTC_fa172.dir/src.c.o] Error 1
make: *** [cmTC_fa172/fast] Error 2


Source file was:
#include <windows.h>
  #include <psapi.h>
  int main(int artc, char** argv) {
    return EnumProcessModules(NULL, NULL, 0, NULL);
  }

Determining if the function shm_open exists in the rt failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-k2TTcl

Run Build Command(s):/usr/bin/make -f Makefile cmTC_04568/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_04568.dir/build.make CMakeFiles/cmTC_04568.dir/build
Building C object CMakeFiles/cmTC_04568.dir/CheckFunctionExists.c.o
/usr/bin/gcc   -fPIC -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wstring-conversion -Wmisleading-indentation -DCHECK_FUNCTION_EXISTS=shm_open  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -MD -MT CMakeFiles/cmTC_04568.dir/CheckFunctionExists.c.o -MF CMakeFiles/cmTC_04568.dir/CheckFunctionExists.c.o.d -o CMakeFiles/cmTC_04568.dir/CheckFunctionExists.c.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-k2TTcl/CheckFunctionExists.c
Linking C executable cmTC_04568
/usr/local/Cellar/cmake/3.25.1/bin/cmake -E cmake_link_script CMakeFiles/cmTC_04568.dir/link.txt --verbose=1
/usr/bin/gcc  -fPIC -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wstring-conversion -Wmisleading-indentation -DCHECK_FUNCTION_EXISTS=shm_open  -Werror=unguarded-availability-new -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names CMakeFiles/cmTC_04568.dir/CheckFunctionExists.c.o -o cmTC_04568  -lrt 
ld: library not found for -lrt
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [cmTC_04568] Error 1
make: *** [cmTC_04568/fast] Error 2



Performing C++ SOURCE FILE Test HAVE_CXX_FLAG_WD654 failed with the following output:
Change Dir: /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-mwlkIz

Run Build Command(s):/usr/bin/make -f Makefile cmTC_af6c2/fast && /Library/Developer/CommandLineTools/usr/bin/make  -f CMakeFiles/cmTC_af6c2.dir/build.make CMakeFiles/cmTC_af6c2.dir/build
Building CXX object CMakeFiles/cmTC_af6c2.dir/src.cxx.o
/usr/bin/g++ -DHAVE_CXX_FLAG_WD654  -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wmisleading-indentation  -std=c++11  -Wall  -Wextra  -Wshadow  -Wsuggest-override  -pedantic  -pedantic-errors  -Wshorten-64-to-32  -fstrict-aliasing  -Wno-deprecated-declarations  -fno-exceptions  -Wstrict-aliasing  -Werror=unguarded-availability-new -wd654 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk   -wd654 -std=c++14 -MD -MT CMakeFiles/cmTC_af6c2.dir/src.cxx.o -MF CMakeFiles/cmTC_af6c2.dir/src.cxx.o.d -o CMakeFiles/cmTC_af6c2.dir/src.cxx.o -c /tmp/flang-1674668575/classic-flang-llvm-project-flang-15.0.3-20221103/build/CMakeFiles/CMakeScratch/TryCompile-mwlkIz/src.cxx
clang: error: unknown argument: '-wd654'
clang: error: unknown argument: '-wd654'
make[1]: *** [CMakeFiles/cmTC_af6c2.dir/src.cxx.o] Error 1
make: *** [cmTC_af6c2/fast] Error 2


Source file was:
int main() { return 0; }
```

--- a/runtime/include/float128.h
+++ b/runtime/include/float128.h
@@ -8,7 +8,7 @@
 #ifndef _FLOAT128_H_
 #define _FLOAT128_H_
 
-#if defined(TARGET_POWER) || (defined(TARGET_X8664) && !defined(TARGET_WIN))
+#if defined(TARGET_LINUX_POWER) || defined(TARGET_LINUX_X8664)
 typedef __float128 float128_t;
 #else
 /* __float128 is not available on AArch64 or other generic targets;

--- a/runtime/libpgmath/lib/common/float128.h
+++ b/runtime/libpgmath/lib/common/float128.h
@@ -16,7 +16,7 @@
 #include <stdint.h>
 
 /* See https://gcc.gnu.org/onlinedocs/gcc/Floating-Types.html. */
-#if defined(LINUX8664) || (defined(TARGET_X8664) && !defined(TARGET_WIN))
+#if defined(TARGET_LINUX_X8664)
 typedef __float128 float128_t;
 typedef _Complex float __attribute__((mode(TC))) quad_complex_t;
 #elif defined(TARGET_LINUX_POWER)

